editor/store-browser-enhancer.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { throttle } from 'lodash';
+
+/**
+ * Enhance a redux store with the browser size
+ *
+ * @param {Object} store Redux Store
+ */
+function enhanceWithBrowserSize( store ) {
+	const updateSize = throttle( () => {
+		store.dispatch( {
+			type: 'BROWSER_RESIZE',
+			width: window.innerWidth,
+			height: window.innerHeight,
+		} );
+	}, 100 );
+
+	window.addEventListener( 'resize', updateSize );
```

Comment:
I’m wondering what will happrn with thise event listeners when Redux store gets recreated after error that crashes editor. It would be great to double check they don’t get cloned every time it happens.